src/imp/platform/browser/options_parser.js

```diff
@@ -9,6 +11,9 @@ let hostScriptElement = (function () {
     if (typeof WorkerGlobalScope !== 'undefined' && self instanceof WorkerGlobalScope) {
         return null;
     }
+    if (!util.isBrowser()) {
```

Comment:
The options_parser module exports two functions, `parseScriptElementOptions` and `parseURLQueryOptions`. It looks to me as though both methods will no-op if `!util.isBrowser()`. Rather than add this conditional, and the one on line 28, can we export functions that no-op instead? It would be similar to how you handled `cookieNoop`, but for `parseScriptElementOptions` and `parseURLQueryOptions`.